site/pages/create-consequence-network/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -240,7 +241,10 @@ export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props:
     );
 
     if (!disruption) {
-        throw new Error("No disruption found for network consequence page");
+        if (ctx.res) {
```

Comment:
Need to add same block for `type-of-consequence` page

Reply:
Also required for `create-disruption` page

Reply:
done

Reply:
not needed on create disruptions it reacts differently